build/docker/alpine.Dockerfile

```diff
@@ -19,11 +19,14 @@ ENTRYPOINT [ "debricked",  "scan" ]
 
 FROM cli AS resolution
 RUN apk --no-cache --update add \
-    openjdk11-jre \
-    python3 \
-    py3-scipy \
-    py3-pip \
-    go~=1.20
+  openjdk11-jre \
+  python3 \
+  py3-scipy \
+  py3-pip \
+  go~=1.20 \
+  nodejs \ 
+  npm \
```

Comment:
Is npm required?

Reply:
NPM is included in nodejs, I'll remove this explicit installation.